pkg/scm/gitlab.go

```diff
@@ -36,6 +37,7 @@ func (gl *Gitlab) Init() error {
 
 	git, err := gitlab.NewOAuthClient(accessToken.Token)
 	gl.Client = git
+	gl.Token = accessToken.Token
```

Comment:
this token should expire pretty quickly, like in a matter of hours

Reply:
a better solve is probably to just do the github star process in-line with plural init or something like that

Reply:
@michaeljguarino  It was my first thought too, but I found this: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/token-expiration-and-revocation#token-expired-due-to-lack-of-use

so the token is valid for one year

Reply:
done